autoscheduler/scraper/models/course.py

```diff
@@ -8,20 +8,14 @@ def generate_course_id(dept: str, course_num: str, term: str):
 class Course(models.Model):
     """ Basic course model """
 
-    id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
+    id = models.CharField(max_length=14, primary_key=True) # Could be dept+course_num

```

Comment:
This will now actually have to be 15, since course_num can be 5 characters long (i.e. LAW 7882S) plus the 6 character term code, plus the 4 character subject/dept.